AmethystAPI/src/minecraft/src-client/common/client/renderer/Tessellator.cpp

```diff
@@ -21,6 +21,20 @@ void Tessellator::vertex(const Vec3& vec) {
     this->vertex(vec.x, vec.y, vec.z);
 }
 
+void Tessellator::vertexUV(float x, float y, float z, float uvX, float uvY)
+{
+    using function = void(Tessellator::*)(float, float, float, float, float);
+    // 1.20.72 - found: xref of Tessellator::vertex(float, float, float)
+    static auto func = std::bit_cast<function>(SigScan("48 83 EC ? 80 B9 ? ? ? ? ? 0F 57 E4"));
+    (this->*func)(x, y, z, uvX, uvY);
+}
+
+
+void Tessellator::vertexUV(const Vec3& vec, float uvX, float uvY)
+{
+    this -> vertexUV(vec.x, vec.y, vec.z, uvX, uvY);
```

Comment:
the spaces in here is a crime against humanity 

I'll fix it up when I head on soon

Reply:
I'm sorr, didn't see it... it looks like this was auto fmt?